### PR TITLE
feat(pyproject): add sane default settings

### PR DESCRIPTION
**Pull request by grski**
for ruff, ruff isort  and remove setup.cfg as it's only used for flake8 which we can remove if we use ruff

### CODE REVIEW

**Comment by CarlosLugones** [comment]
Nice! I had no idea this was possible with Poetry.

**Comment by grski** [comment]
@CarlosLugones it comes bundles with ruff, pyproject is just a standard way of config that Poetry promoted 👍  and thank you!

File: {{cookiecutter.repo_name}}/pyproject.toml [hunk 1]
```diff
@@ -17,15 +17,61 @@ psycopg2-binary = "^2.9.6"
...
+    "__pypackages__",
+    "buck-out",
+    "build",
+    "dist",
+    "node_modules",
+    "venv"
+]
+
+[tool.ruff.isort]
+section-order = ["fastapi", "future", "standard-library", "third-party",  "first-party", "local-folder"]
```
Comment (adamjanas) [inline {{cookiecutter.repo_name}}/pyproject.toml:70]: @grski place future imports right before `fastapi` (as first)

File: {{cookiecutter.repo_name}}/pyproject.toml [hunk 2]
```diff
@@ -17,15 +17,61 @@ psycopg2-binary = "^2.9.6"
...
+  | \.git
+  | \.hg
+  | \.mypy_cache
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+  | migrations
+)/
```
Comment (Lolomgrofl) [inline {{cookiecutter.repo_name}}/pyproject.toml:43]: exclude `alembic` here as well

**Review by Lolomgrofl — CHANGES_REQUESTED** [review]

**Comment by grski** [comment]
@Lolomgrofl @adamjanas resolved.

**Comment by Lolomgrofl** [comment]
@grski pre-commit failed

**Comment by grski** [comment]
Finally.
Should be good.

**Review by Lolomgrofl — APPROVED** [review]

**Pull request merged**